css-three-column-layout/styles.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  background-color: rgb(25 150 218);
+  color: white;
+  font-style: normal;
+  font-weight: 400;
+  margin: 0;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+}
+
+.column-one-third {
+  display: inline-block;
```

Comment:
This shouldn't be needed if you were using flexbox. Since you don't have `display: flex` on your row, you aren't actually using flexbox here.